src/components/forms/PlanForm/index.tsx

```diff
@@ -1061,7 +1061,7 @@ const PlanForm = (props: PlanFormProps) => {
               className="btn btn-block"
               color="primary"
               aria-label={SAVE_PLAN}
-              disabled={isSubmitting || Object.keys(errors).length > 0}
+              disabled={isSubmitting || Object.keys(errors).length > 0 || !isValid}
```

Comment:
@ciremusyoka question? in what way is `Object.keys(errors).length > 0` different from `!isValid`

Reply:
@p-netm from my observations sometimes before any field is touched `errors` are an empty object. That is why i decided to use the two checks.

Reply:
:+1: